MonoGame.Framework/SDL/SDL2.cs

```diff
@@ -0,0 +1,717 @@
+﻿// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+using System;
+using System.Text;
+using System.Runtime.InteropServices;
+
+internal static class Sdl
+{
+    private const string NativeLibName = "SDL2.dll";
```

Comment:
> Can one of you modify the targets file? 

If you can give me a day or two i can...  just need to figure out how to detect 64bit vs 32bit.

Maybe this shouldn't hold up this PR however?

@dellis1972 @KonajuGames - Do you think this can be merged for now without the DesktopGL template adding the correct SDL2 assembly for the user?